doc/user/assets/sass/_content.scss

```diff
@@ -218,6 +218,13 @@ a {
     padding: 16px;
   }
 
+  .note {
```

Comment:
nit: users often _avoid_ reading "note" text in docs, which is why there wasn't an existing style. I used warning hesitantly but don't recommend notes in general. That being said, whatever you think is best.

Reply:
I was surprised that we didn't have more admonitions available. 😛  I see it more as a way of breaking the text, to be honest; while going through the docs, I missed notes that could be relevant a couple of times just because the content looks so homogenous. I'd keep it and take the blame if someone roadrages at them!